src/components/MapInput/__tests__/MapInput.test.js

```diff
@@ -188,7 +177,10 @@ describe('components/MapInput', () => {
         docs: [],
       },
     };
-    fetch.mockResponseOnce(JSON.stringify(noneFoundResponse)).mockResponseOnce(JSON.stringify(bagResponse));
+    fetch
+      .mockResponseOnce(JSON.stringify(noneFoundResponse))
+      .mockResponseOnce(JSON.stringify(geocoderResponse))
+      .mockResponseOnce(JSON.stringify(bagResponse));
```

Comment:
This line can go as well since there is no request and response to and from the BAG API

Reply:
:+1: